core/ecschema-editing/src/Editing/CustomAttributes.ts

```diff
@@ -43,6 +43,17 @@ export class CustomAttributes extends ECClasses {
     return { itemKey: newClass.key };
   }
 
+  public async delete(schemaKey: SchemaKey, name: string): Promise<SchemaItemEditResults> {
+    const schema = await this._schemaEditor.getSchema(schemaKey);
```

Comment:
I count 5 identical-looking copies of this function...possible to consolidate them?

Reply:
I moved delete function to ECClasses so there is only one copy of this.